src/function/pattern/label_function.cpp

```diff
@@ -74,27 +74,32 @@ std::shared_ptr<Expression> LabelFunction::rewriteFunc(const RewriteFunctionBind
         return expressionBinder->bindScalarFunctionExpression(children,
             StructExtractFunctions::name);
     }
+    auto disableLiteralRewrite = expressionBinder->getConfig().disableLabelFunctionLiteralRewrite;
```

Comment:
Can we not expose the ExpressionBinderConfig?
We can expose an api in expressionBinder like: `expressionBinder->allowLabelFunctionRewrite`?

Reply:
Ditto